src/Concerns/ManagesTransactions.php

```diff
@@ -84,7 +84,7 @@ public function transaction(Closure $callback, $attempts = -1)
                     $this->transactions++;
 
                     $this->transactionsManager?->begin(
-                        $this->getName(), $this->transactions,
+                        $this->getName() ?? '', $this->transactions,
```

Comment:
This changes the behavior of the unerlying code.
Before -> error if null.
After -> no error.

This should be a separate PR and it should throw an error.

Reply:
Initially I was throwing errors, but I stopped because the layers where the errors occur change.

Reply:
Fixing in another PR sounds good. Let's just add a ignore here for now.